provers/cairo/cairo_compile.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends g++ libgmp3-dev
+
+COPY requirements.txt /
```

Comment:
it seems that the requirements file is missing, I got an error when building the docker image